inicio.js

```diff
@@ -6,52 +6,63 @@ import {tags} from "@lezer/highlight"
 import {javascript} from "@codemirror/lang-javascript"
 import {keymap, KeyBinding} from "@codemirror/view"
 import { SptmAudio } from "./controlador"
+import { AudioSetup, Sine, Noise } from "./audioSetup"
 
 const { 
-    init,
-    start,
-    startSine,
-    suspend,
-    reproducirRuidoFunc,
-    detenerRuidoFunc,
-    reproducirSineFunc,
-    detenerSineFunc,
-    reproducirMicFunc,
-    detenerMicFunc,
     encenderCamaraFunc,
     detenerCamaraFunc,
     iniciarAF1,
     iniciarAF2,
-    chgainNoise,
-    chgainSine
 } = SptmAudio()
 
-start()
-startSine()
+let a = new AudioSetup;
```

Comment:
aqui parece que estas inicializando un objeto, entonces ya tiene sentido el "this" que usaste anteriormente, quiza sería buena idea utilizar la notacion de clases de ES6 para evitar confusiones
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes